query/src/configs/config_utils.rs

```diff
@@ -0,0 +1,12 @@
+/// Mask a string by "******". If the lenght of the string is equal or greater than 9, keep the 3-byte suffix.
+#[inline]
+pub fn mask_string(s: &str) -> String {
```

Comment:
Would be better move `mask_string` function to https://github.com/datafuselabs/databend/blob/main/common/base/src/string_func.rs?

Reply:
To make the function more general:
`pub fn mask_string(s: &str, reveal_num: int)`
mask_string("abcdefg", 2) -> `****fg` 
mask_string("abcdefg", 0) -> `******`